pkg/cluster/internal/providers/podman/provider.go

```diff
@@ -186,7 +186,10 @@ func (p *provider) GetAPIServerEndpoint(cluster string) (string, error) {
 	if err != nil {
 		return "", errors.Wrap(err, "failed to check podman version")
 	}
-	if v.LessThan(version.MustParseSemantic("2.2.0")) {
+	// podman inspect was broken between 2.2.0 and 3.0.0
+	// https://github.com/containers/podman/issues/8444
+	if v.LessThan(version.MustParseSemantic("2.2.0")) ||
```

Comment:
nit: let's invert the logic and have the "hack" inside the `if` and the actual flow to be the normal flow.

do we also want to output a warning?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow +1